src/clients/java/src/test/java/com/tigerbeetle/IntegrationTest.java

```diff
@@ -2186,6 +2187,33 @@ public void testImportedFlagAsync() throws Throwable {
         assertEquals(timestamp + accounts.getLength() + 1, lookupTransfers.getTimestamp());
     }
 
+    /**
+     * Asserts that empty replies are not shared.
+     * https://github.com/tigerbeetle/tigerbeetle/pull/2495
+     */
+    @Test
+    public void testEmptyReply() throws Throwable {
+        final int TASKS_COUNT = 1_000;
```

Comment:
If we are going to add this to the integration test suite, let's change the task count to 4 or so... it didn't take that many repetitions to hit the bug, and I don't want to make the test suite slower :smile:

Reply:
Cool, it's about 300ms now!